### PR TITLE
Refs 170: changed plugin name to "content_sources"

### PR DESCRIPTION
**Pull request by mayurilahane**

### CODE REVIEW

**Review by mshriver — APPROVED** [review]

**Comment by mshriver** [comment]
holding draft, we'll merge this after the name change is pushed elsewhere. ci.int pipeline will fail until the newly named containers are built.

@mayurilahane all PRs need their title to start with `Refs <number>`, a reference to the related `HMSCONTENT-<number>` jira card.

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-170

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

**Pull request merged**
